katalog/curator-s3/README.md

```diff
@@ -0,0 +1,55 @@
+# Curator S3
+
+Curator helps you manage your Elasticsearch indices and snapshots via various
+operations like delete, snapshot, and shard allocation routing. It's mainly used
+to managed retention of your infrastructure logs to a given value.
+
+## Requirements
+
+- It requires elasticsearch with installed the plugin `repository-s3`
+- an object storage bucket aws s3 compatible
+- Kubernetes >= `1.17.0`
+- Kustomize >= `v3`
+
+## Image repository and tag
+
+- The Dockerfile of elasticsearch should be like [this](elasticsearch/Dockerfile) or
+  a manifest with an initcontainer that setup the same logic
+- Curator image: `quay.io/sighup/curator:5.8.3_3.8-alpine_3.13`
+- Curator repo: [https://github.com/elastic/curator](https://github.com/elastic/curator)
+- Curator documentation:
+  [https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html)
+
+## Configuration
+
+- Replica number: `1`
+- Unit set as `30 days`
```

Comment:
nice question @ralgozino , in the past I used the bucket lifecycle rule option that you can set when you create it, but I think that the best option should be to add a cronjob that deletes consistently the selected index from the bucket. This because of course, the lifecycle rule is not aware of the structure of the index and can leave broken indexes. 
I've to add it before release it